src/components/Home.test.js

```diff
@@ -1,17 +1,78 @@
 import React from 'react';
 import { shallow } from 'enzyme';
-import Home from './Home';
+import { HomeComponent, HomeButtonsMajor, HomeButtonsMinor } from './Home';
+import { Link } from 'react-router-dom';
+Link.displayName = 'Link';
+jest.mock('@material-ui/core/Button', () => 'Button');
```

Comment:
We should find and/or make something that does all these material mocks with one import

Reply:
Not for this PR but just in general

Reply:
@rhinodavid could you make an issue for that so we remember to address it?

Reply:
check the jest docs on mocking node_modules., i think you could either set up a mock's file and it'll generate mocks for you, or there is a factory arg to the jest.mock function. not sure, but that may help.